src/regressor.rs

```diff
@@ -344,12 +345,21 @@ impl Regressor {
         pb.reset(); // empty the tape
 
         let further_blocks = &self.blocks_boxes[..];
-        block_helpers::forward(further_blocks, fb, pb);
+        let mut predictions_sum: f32 = 0.0;
+        // We call once using all interactions
+        block_helpers::forward(further_blocks, fb, pb, false);
 
-        assert_eq!(pb.observations.len(), 1);
-        let prediction_probability = pb.observations.pop().unwrap();
+        // And then 29 times masking a portion of the interactions
+        for n in 1..30 {
```

Comment:
Let's parametrize this at the top of the file at least (if not via cli) .. 30 seems arbitrary + most likely too much

Reply:
This will be very costly when running multiple times (even if we skip multiplications)

Reply:
@ggaspersic I've discussed with Karni some ideas of computing performance improvements, but it requires a few more hours of work (involves changing the computational graph and only running the multiple predictions in the ffm part)..
but for now we will have this implementation as a baseline, because running the whole graph multiple times might also affect the model performance, so this might be worse in terms of computation, but will be good later to see that montecarloing only ffm reaches similar model performance

Reply:
@SkBlaz in the last commit I've added both the number of monte carlo predictions and the dropout rate to the cmdline